GNUmakefile

```diff
@@ -489,6 +507,24 @@ dist_libstemmer_js: $(JS_SOURCES)
 	(cd dist && tar zcf $${destname}$(tarball_ext) $${destname}) && \
 	rm -rf $${dest}
 
+dist_libstemmer_ts: $(TS_SOURCES)
+	destname=tsstemmer-$(SNOWBALL_VERSION); \
+	dest=dist/$${destname}; \
+	rm -rf $${dest} && \
+	rm -f $${dest}$(tarball_ext) && \
+	mkdir -p $${dest} && \
+	mkdir -p $${dest}/$(ts_runtime_dir) && \
+	mkdir -p $${dest}/$(ts_sample_dir) && \
+	cp -a doc/libstemmer_ts_README $${dest}/README.rst && \
```

Comment:
There's no `doc/libstemmer_ts_README` so `make dist` fails (hence the CI failures)